lego_ee_tf: "/transforms/lego_ee.tf"
lego_world_tf: "/transforms/lego_world_2.tf"
tmp_lego_world_tf: "/transforms/tmp_lego_world.tf"
data_dir: "/mnt/hdd1/vibrotactile_data/lego/"
x_range:
  - -0.005
  - 0.005
y_range: 
  - -0.005
  - 0.005
theta_range: 
  - -10.0
  - 10.0
block_x_range: 
  - -15
  - 0
block_y_range:
  - 0
  - 10
approach_height: 0.03
lift_height: 0.01
pull_up_velocity_scaling: 0.001
place_rotation: 15
pick_rotation: 15
fts_detector:
  topic_name: '/fts'
  force_threshold: 0.4
lego_detector:
  topic_name: '/side_camera/color/image_cropped'
  top_bbox:
    - 100
    - 250
    - 1200
    - 400
  bot_bbox:
    - 100
    - 390
    - 1200
    - 700
  detection_threshold: 0.9
rosbag_data_recorder:
  topics:
    - /side_camera/color/image_cropped
    - /camera/color/image_raw
    - /namespace/joint_states
    - /fts
    - /audio
    - /audio_info
    - /namespace/terminator/skill_termination_signal
    - /outcome/lego_detector
    - /outcome/fts_detector
    - /namespace/skill/param
    - /namespace/terminator/termination_config